index.html

```diff
@@ -0,0 +1,332 @@
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic2.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
+                            <div class="codingLangButtons">
+                                <div class="codingLangBackground">
+                                    HTML/CSS
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic3.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
+                            <div class="codingLangButtons">
+                                <div class="codingLangBackground">
+                                    HTML/CSS
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
```

Comment:
- [ ] kindly consider making this  a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design) to implement the user interactions as given in the design below  kindly use any of  your choices
 
![one](https://user-images.githubusercontent.com/60146030/221381841-f53ddbcd-036e-4354-ad4e-5e783c48d0d7.PNG)